qa/rpc-tests/test_framework/authproxy.py

```diff
@@ -44,7 +44,7 @@
 
 HTTP_TIMEOUT = 600
 
-log = logging.getLogger("BitcoinRPC")
+log = logging.getLogger("ZcashRPC")
```

Comment:
Probably not correct.

Reply:
Reverted (to make logs consistent over time?).